expr/functions/linearRegression/function.go

```diff
@@ -87,7 +88,9 @@ func (f *linearRegression) Do(ctx context.Context, e parser.Expr, from, until in
 		for i := range r.Values {
 			r.Values[i] = consolidations.Poly(float64(i), c.RawMatrix().Data...)
 		}
-		results = append(results, &r)
+		r.Tags["linearRegressions"] = fmt.Sprintf("%d, %d", a.GetStartTime(), a.GetStopTime())
```

Comment:
Aso, may be without Sprintf ?

Reply:
Sounds good. I'll try to address this and all your other comments related to Sprintf